.github/CODEOWNERS

```diff
@@ -1 +1 @@
-*   @TrueCloudLab/dev
+*   @TrueCloudLab/storage-core @TrueCloudLab/storage-services @TrueCloudLab/committers
```

Comment:
Why github marks storage-services team as unknown owner? Is it github issue?
![image](https://user-images.githubusercontent.com/125876766/222061122-79086726-2a62-44a4-93a5-fd86ec281751.png)

Reply:
think they do not have write access in that repo

Reply:
HRW was identified as Storage Core group's responsibility, hence Storage Services group has no direct write permission.